packages/flutter/test/material/search_test.dart

```diff
@@ -129,10 +129,10 @@ void main() {
     ));
     await tester.tap(find.byTooltip('Search'));
     await tester.pumpAndSettle();
-
-    final TextField textField = tester.widget<TextField>(find.byType(TextField));
-    final Color hintColor = textField.decoration!.hintStyle!.color!;
-    expect(hintColor, delegate.hintTextColor);
+    
+    final ThemeData? textFieldTheme = Theme.of(tester.element(find.byType(TextField)));
```

Comment:
Ooh, I didn't know that getting the text directly would also get the style like that. I changed to match the `InputDecorationTheme` check because `hintStyle` was returning null (since we're applying a theme now instead of a direct `InputDecoration`). Will update the test, thanks.